### DISCUSSION TITLE
Vaultwarden Ngnix Proxy - CloudFlare reverse proxy WEBAUTH not working

### DISCUSSION THREAD

**Discussion by DavidCroitoru** [dani-garcia/vaultwarden#4997] [Q&A]
I am trying to add a yubico key to my vault for 2fa but i keep geeting an error ,,There was a problem reading the security key. Try again." . I have added the DOMAIN env variable acording to my website. I dont have a self sign cert with Let's Encrypt on the local host. I have my istance on docker. 

I have added a console log screenshot. 

Thanks!

**Comment by BlackDex** [comment]
Looks like you didn't configured the domain correctly

**Reply by DavidCroitoru** [reply]
In the docker envs?

![image](https://github.com/user-attachments/assets/e209eab8-52ea-4075-b987-5904c016ec07)

**Reply by BlackDex** [reply]
I would suggest to remove the port and see what happens.

**Reply by DavidCroitoru** [reply]
I have already tried without the port. I have added the port to see if it would work. 

Domain configuration No Match No HTTPS
Server: http://localhost/admin/diagnostics
Browser: https://vault.da..........ro/admin/diagnostics

**Reply by BlackDex** [reply]
How come your server thinks it localhost as the domain configured then? Something is wrong in your config.

**Reply by DavidCroitoru** [reply]
The domain config its just by the DOMAIN env, right?

**Reply by stefan0xC** [reply]
Not if you have a `data/config.json`. Cf. https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview

**Reply by BlackDex** [reply]
It is as easy as to going to `/admin/diagnostics` if it reports the serer as you posted above, the DOMAIN is not configured correctly. You probably want it to match what the browser reports, and that will also result in an `OK` message.

**Reply by DavidCroitoru** [reply]
Yes, worked i copied the domain name from the diagnostiscs page and pasted it into the config. I put a / in the end of the url. Thanks!
